extensions/blocks/rating-star/style.scss

```diff
@@ -0,0 +1,101 @@
+// Editor and View styles
+.wp-block-jetpack-rating-star,
+.wp-block-jetpack-rating-spiciness,
+.wp-block-jetpack-rating-priciness {
+	margin-bottom: 1.5em; // Provide a minimum bottom margin.
```

Comment:
You can pull this from a variable:
```scss
@import '../../shared/styles/jetpack-variables.scss';

margin-bottom: $jetpack-block-margin-bottom;
```

Reply:
Now we know the final place where ratings will live, I guess is proper to start using the variables defined in that place. Done at https://github.com/Automattic/jetpack/pull/14091/commits/45aac1b6bd9934fc5574db4855a8e7896da158d4